server/src/main/java/io/spine/server/entity/model/EntityClass.java

```diff
@@ -57,6 +57,10 @@
     /** Type of the entity state. */
     private final TypeUrl entityStateType;
 
+    /** The helper for working with ID field of the entity state. */
```

Comment:
It's an `IdFieldManager` than?

Reply:
I'd rather avoid calling things `Manager` as much as possible. You allow it once and there's no way to stop it after...